src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -174,9 +176,14 @@ def _convert_assets(self, network: Network) -> List[AssetAbstract]:
             # Esdl Junctions need to be skipped for now, are added later.
             if isinstance(esdl_asset.esdl_asset, esdl_junction):
                 continue
-            py_assets_list.append(EsdlAssetMapper.to_entity(esdl_asset))
-            py_assets_list[-1].add_physical_data(esdl_asset=esdl_asset)
-            network.add_existing_asset(py_assets_list[-1].solver_asset)
+            if isinstance(esdl_asset.esdl_asset, esdl_Pipe):
```

Comment:
This should not be changed in this PR. You can still call the ESDLAssetMapper, since in there you have implemented the logic of selecting to what type it needs to be converted. We can discussed but in different issue if we want to change this. So please reverted these change.

Reply:
Completed.